### ISSUE TITLE
[DK-UH] Will Death Knight Unholy's rotation be updated?

### ISSUE THREAD

**Issue by Dragonfo** [herotc/hero-rotation#652]
It seems to me that now the unholy rotation does not work exactly according to APL. And it was not updated for a very long time.

**Comment by azzi1103** [comment]
BUMP

**Comment by Dragonfo** [comment]
Who else can renew our rotation? Much time has passed. And nobody even answered.

**Comment by zaberwookie** [comment]
i main uh dk just use an older version of hero rotation and tellmewhen addon for new stuff thats missing for now

**Comment by zaberwookie** [comment]
HeroRotation-8.3.0.00 is what i use

**Comment by Dragonfo** [comment]
You probably are not confused by the fact that there is no synergy and consistency of abilities in the rotation. Actually, this is how we have a legendary power, and we play through it differently, plus the covenant and it also needs to be put into rotation at the right time. If you played on dk you would know that. And I would not advise nonsense. I'm waiting for someone who can fix the situation at dk. And not to advise the outdated rotation.

**Comment by zaberwookie** [comment]
> You probably are not confused by the fact that there is no synergy and consistency of abilities in the rotation. Actually, this is how we have a legendary power, and we play through it differently, plus the covenant and it also needs to be put into rotation at the right time. If you played on dk you would know that. And I would not advise nonsense. I'm waiting for someone who can fix the situation at dk. And not to advise the outdated rotation.

With some brain power, research and maybe some skill you can have synergy, better than not playing with training wheels i guess? 

I use icy veins https://www.icy-veins.com/wow/unholy-death-knight-best-legendaries-shadowlands, bloodmallet https://bloodmallet.com/ and wowhead websites to do research on proper rotation. I put the necro ability (and other abilities/trinkets) into my rotation with TellMeWhen addon showing me when the ability is available or how long until I can use it situationally like during a big trash pull or boss since its a 2min cd. Once I craft my *overall bis* legendary https://www.wowhead.com/item=182638/memory-of-a-frenzied-monstrosity that drops from a World Boss that hasn't even been released yet as of this writing 12/13/2020. https://www.wowhead.com/npc=167526/nurgash-muckformed hopefully next Tuesday. I heard its smarter to farm the ash and wait till you can afford to upgrade straight to rank 4 legendary, most will just start with the base legendary at the very least. Rumors that they will allow 2 legendaries in the next big content patch like in Legion as well.

**Comment by azzi1103** [comment]
icy veins is a joke, so is bloodmallet. Use wowhead and herodmg to check out proper class guide. Geniuses like mystler are there. The overall bis for UH is the death coil legendary. It keeps your Dark transformation up longer, therefore giving you way longer cleave/burst time. And it has a synergy with Eternal Hunger which frenzied monstrosity doesn't have.

**Comment by zaberwookie** [comment]
> icy veins is a joke, so is bloodmallet. Use wowhead and herodmg to check out proper class guide. Geniuses like mystler are there. The overall bis for UH is the death coil legendary. It keeps your Dark transformation up longer, therefore giving you way longer cleave/burst time. And it has a synergy with Eternal Hunger which frenzied monstrosity doesn't have.

I mean if I was a true CE elitist jerk I wouldn’t even need addons or websites to tell me how ‘git gud scrub’ Iam lol most people who use rotation addons need a little extra help figuring things out or are just too lazy/sane to go balls deep spergy with the toxic min/max culture.

I’m somewhere in-between, I’ve had past ‘hard core raider’ experience in wrath when I got immortal/undying titles and black,plague and iron bound proto drakes during the meta back than. Than my most recent ‘hArDcOrE tOxiC’ achievement was receiving ‘The Chosen’ title in legion. Than bfa came out and it sucked allot that’s when I decided to just go super casual and stop stressing over pixels. 

I started playing wow because all my irl LAN party counter strike buddies switched to human mages and wanted me to pocket heal them back in 04’. I refused but finally gave in April 05 lol. 

Since burning crusade I went shadow and never looked back until they messed up and kept nerfing spriests back and forth. So I switched to DH in legion, got sick of that now I main dk,pally and warrior. All the plate nice contrast to squishy casters. Soon an mm hunter. Maybe I’ll level my spriest again. She has the benediction/anathema to prove how ‘old og hardcore respect my authority’ Iam ig lol

Coupled with yummy salty unobtainables like  black proto, the og brew fest mount, ‘champion of the naru’, ‘hand of a’dal’, ‘the immortal’, ‘the undying’, ‘the chosen’ and even ‘field medic’ title haha. People get real salty real quick it’s so obvious especially when they deny it. I love it totally worth all those wasted years of my life, almost. 

It’s not worth over working yourself over a video game I’ve learned. Or anything really. If something’s really stressing you out just take a break take it easy. Stress is the true killer
